pyronear/datasets/wildfire/wildfire.py

```diff
@@ -191,3 +193,97 @@ def set_splits(self, dataframes):
         # Determine estimated(posterior) parameters
         self.n_samples_ = {set_: len(self.splits[set_]) for set_ in ['train', 'val', 'test']}
         self.ratios_ = {set_: (self.n_samples_[set_] / len(self.wildfire)) for set_ in ['train', 'val', 'test']}
+
+
+class WildFireSubSampler:
+    """Sub-Sample a WildFireDataset
+
+    The Wildfire DataSet is annotated with video sequences. To extract this dataset as frames,
+    you have to choose the number N of frames per sequence. The idea of this SubSampler is to
+    combine K frames among these N to create a derivative of the resnet the ssresnet.
+
+    Parameters
+    ---------
+    metadata: Pandas.DataFrame
+        metadata of the WilFireDataset
+
+    frame_per_seq: int
+        frame per sequence to take
+
+    frame_per_seq: float
+        between 0 and 1, percentage of non-fire data to take twice in order to equalize the dataset
+
+    Example
+    -------
+    subsampler = WildFireSubSampler(dataset.metadata, 2)
+    dataset.metadata = subsampler.computeSubSet()
+
+    """
+    def __init__(self, metadata, frame_per_seq, probTh=None):
+
+        self.metadata = metadata
+        self.metadata.index = np.arange(len(self.metadata))
+        self.imgs = self.metadata['imgFile']
+        self.SubSetImgs = []
+        self.probTh = probTh
+        self.frame_per_seq = frame_per_seq
+        # Define sequences numbers
```

Comment:
Why are you re-extracting the fBase from the imgFile ? Shouldn't it be already present since previous step was Frame extraction  ?